Update container-instances-mounting-azure-files-volume.md

bureado opened the pull request:
Fixed $ in assignment
Removed hard coded resource group and storage account names

azurecla commented:
Hi __@pcgeek86__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, AZPRBOT;

Selcin commented:
Hi Trevor,
Thank you for your contribution. I reviewed your feedback and just updated the article based on your feedback with a slightly different wording. It should be live early next. If you have any other feedback, please feel free to let us know. Thanks again!

bureado committed:
Update container-instances-mounting-azure-files-volume.md

Fixed $ in assignment
Removed hard coded resource group and storage account names
